compiler/test/fail_compilation/ufcs.d

```diff
@@ -0,0 +1,31 @@
+/*
+TEST_OUTPUT:
+---
+fail_compilation/ufcs.d(22): Error: no property `regularF` for type `S`, UFCS rewrite failed because:
```

Comment:
> the following errors occured while looking for a UFCS match

"UFCS rewrite failed because" is ugly

Reply:
I tried to keep it concise so it fits on 1 line, should I make it 2 lines?

Reply:
Yes.

Reply:
Done